cdk/v2/utils.js

```diff
@@ -100,9 +110,21 @@ function getErrorInfo(err) {
   return errorInfo;
 }
 
+function isCdkV2Destination(event) {
+  return Boolean(
+    event?.destination?.DestinationDefinition?.Config?.cdkV2Enabled
+  );
+}
+
+function getCdkV2TestThreshold(event) {
```

Comment:
can we rename `cdkV2Test` to `cdkV2TestThreshold` ?

Reply:
Sure

Reply:
Done

Reply:
Done resolving.